app/src/main/java/practicum/getfitla_v3/ExerciseItemFormat.java

```diff
@@ -7,11 +7,11 @@
     private int id;
     private String title;
     private String shortdesc;
-    private double rating;
-    private double price;
+    private String rating;
+    private String price;
     private int image;
 
```

Comment:
Add a description to the top of this file as well.

Reply:
There is already a comment at the top, just not shown in this review